projects/1.18/assets/skyopolis-mod/skyopolis/lang/zh_cn.json

```
@@ -0,0 +1,39 @@
+{
+  "item.skyopolis.leafy_string": "叶状线",
+  "item.skyopolis.organic_seeds": "有机种子",
+  "item.skyopolis.wooden_shears": "木剪刀",
+  "item.skyopolis.andesite_pebble": "安山岩碎块",
+  "item.skyopolis.granite_pebble": "花岗岩碎块 ",
+  "item.skyopolis.diorite_pebble": "闪长岩碎块",
+  "item.skyopolis.stone_pebble": "石头碎块",
+  "item.skyopolis.gravel_dust": "沙砾粉",
+  "item.skyopolis.sand_dust": "沙子粉",
+  "item.skyopolis.grout_dust": "水泥粉",
+  "item.skyopolis.activated_spawner_shard": "激活的孵化碎片",
+  "item.skyopolis.spawner_shard": "孵化碎片",
+  "item.skyopolis.mushroom_ball": "蘑菇球",
+  "item.skyopolis.waste": "废料",
+  "item.skyopolis.aqua_catalyst": "水催化剂",
+  "item.skyopolis.blank_pebble": "原矿碎块",
+  "item.skyopolis.blank_totem": "白色不死图腾",
+  "item.skyopolis.blank_dust": "原矿粉",
+  "item.skyopolis.blank_nugget": "原矿粒",
```

有同上的石子尘土翻译问题，还有这不是不死图腾
```suggestion
  "item.skyopolis.blank_pebble": "空白石子",
  "item.skyopolis.blank_totem": "空白图腾",
  "item.skyopolis.blank_dust": "空白尘土",
  "item.skyopolis.blank_nugget": "空白粒",
```